eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -181,22 +181,25 @@ foreach ($service in $serviceNameList) {
   $clientPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('client' -eq $_.Type) })
   $clientPackages = $clientPackages | Sort-Object -Property Package
   foreach ($clientPackage in $clientPackages) {
-    $packageItems += GetClientPackageNode -clientPackage $clientPackage
+    $packageItems += GetPackageNode -clientPackage $clientPackage
   }
 
   # All management packages go under a single `Management` header in the ToC
   $mgmtPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('mgmt' -eq $_.Type) })
   $mgmtPackages = $mgmtPackages | Sort-Object -Property Package
-  if ($mgmtPackages) {
-    $children = &$GetDocsMsTocChildrenForManagementPackagesFn `
-      -packageMetadata $mgmtPackages `
-      -docRepoLocation $DocRepoLocation
-
+  $serviceReadmeBaseName = $service.ToLower().Replace(' ', '-').Replace('/', '-')
+  $mgmtItems = @()
+  foreach ($pkg in $mgmtPackages) {
+    # $children = &$GetDocsMsTocChildrenForManagementPackagesFn `
```

Comment:
Is this function going away or do you plan to call it again?

Reply:
This function is used in the Java implementation to fetch namespaces from Preview and GA packages (covering the case that a package is introducing new namespaces we want to show all namespaces in the relevant ToC) -- https://github.com/Azure/azure-sdk-for-java/blob/main/eng/scripts/docs/Docs-ToC.ps1#L54

Reply:
I planed to remove. The PR is in early stage. Will focus this PR once service level readme is ready